src/main/scala/com/cognite/sdk/scala/v1/files.scala

```diff
@@ -59,3 +59,20 @@ final case class FilesQuery(
     search: Option[FilesSearch] = None,
     limit: Int = 100
 ) extends SearchQuery[FilesFilter, FilesSearch]
+
+sealed trait FileDownload
+final case class FileDownloadId(id: Long = 0) extends FileDownload
```

Comment:
Does it make sense for this to have a default value of 0? In the case of `File` we use it to allow easy conversion to `FileCreate` (where `id` is optional).